python-sdk/tests/files/locations/test_wasb.py

```diff
@@ -43,3 +45,26 @@ def test_wasb_smartopen_uri_wasbs_scheme():
 def test_wasb_smartopen_uri_azure_scheme():
     location = WASBLocation(path="azure://somepath")
     assert location.smartopen_uri == "azure://somepath"
+
+
+def test_snowflake_stage_path_raise_exception():
+    """
+    Test snowflake_stage_path raise exception when 'storage_account' is missing.
+    """
+    location = WASBLocation(path="azure://somepath")
+    error_message = f"Required param missing 'storage_account', pass {location.LOAD_OPTIONS_CLASS_NAME}"
+    "(storage_account=<account_name>) to load_options"
```

Comment:
```suggestion
    error_message = (
        f"Required param missing 'storage_account', pass {location.LOAD_OPTIONS_CLASS_NAME} "
        f"(storage_account=<account_name>) to load_options"
    )
```
look like f is missing here

Reply:
@pankajastro It's just an example with a placeholder `<account_name>`. I didn't wan this to be a fstring.

Reply:
I see but it does not look right if we beak a long string in multiline half of them are f-string and half are not. even in this PR you have f-string without parameter https://github.com/astronomer/astro-sdk/pull/1675/files#diff-14125a15a8984cccdb2681309d0e24b1ae1800474ac05a3c319264a8446b36c0R107